src/htmx.js

```diff
@@ -1141,31 +1141,51 @@ var htmx = (function() {
    */
   function querySelectorAllExt(elt, selector, global) {
     elt = resolveTarget(elt)
-    if (selector.indexOf('closest ') === 0) {
-      return [closest(asElement(elt), normalizeSelector(selector.substr(8)))]
-    } else if (selector.indexOf('find ') === 0) {
-      return [find(asParentNode(elt), normalizeSelector(selector.substr(5)))]
-    } else if (selector === 'next') {
-      return [asElement(elt).nextElementSibling]
-    } else if (selector.indexOf('next ') === 0) {
-      return [scanForwardQuery(elt, normalizeSelector(selector.substr(5)), !!global)]
-    } else if (selector === 'previous') {
-      return [asElement(elt).previousElementSibling]
-    } else if (selector.indexOf('previous ') === 0) {
-      return [scanBackwardsQuery(elt, normalizeSelector(selector.substr(9)), !!global)]
-    } else if (selector === 'document') {
-      return [document]
-    } else if (selector === 'window') {
-      return [window]
-    } else if (selector === 'body') {
-      return [document.body]
-    } else if (selector === 'root') {
-      return [getRootNode(elt, !!global)]
-    } else if (selector.indexOf('global ') === 0) {
-      return querySelectorAllExt(elt, selector.slice(7), true)
-    } else {
-      return toArray(asParentNode(getRootNode(elt, !!global)).querySelectorAll(normalizeSelector(selector)))
+
+    const parts = selector.split(',')
+    const result = []
+    const unprocessedParts = []
+    for (let i = 0; i < parts.length; i++) {
+      let selector = parts[i].trimStart()
```

Comment:
trim() is pobably better here as trailing spaces could cause issues as well for 'closest ', 'global ', 'find ' etc.  trailing spaces should never really be passed in but why not deal with them anyway just in case anyway.

Reply:
Actually we could use `normalizeSelector(parts[i])`  This does a trim and some additional handling and will allow us to handle `<input/>, <button/>` which would not be handled well otherwise as it will strip to `input/>, <button`.  we could then remove normalizeSelector from the last unprocessed parts query i think but may still need it for the middle 4 cases for someone doing `find <input/>`

Reply:
Indeed, forgot about `normalizeSelector` which is literally above this function 🤦‍♂️  thanks!